mapper/complementaryContentMapper.go

```diff
@@ -3,11 +3,19 @@ package mapper
 import (
```

Comment:
Organize the imports the following way https://github.com/golang/go/wiki/CodeReviewComments#imports.